src/main/java/appeng/core/sync/network/PacketLogger.java

```diff
@@ -19,13 +19,34 @@
 package appeng.core.sync.network;
 
 
-import net.minecraft.entity.player.EntityPlayer;
+import io.netty.handler.logging.LogLevel;
+import io.netty.handler.logging.LoggingHandler;
 
-import cpw.mods.fml.common.network.internal.FMLProxyPacket;
+import appeng.core.sync.AppEngPacket;
 
 
-public interface IPacketHandler
+public class PacketLogger extends LoggingHandler
 {
+	private String side;
 
-	void onPacketData( INetworkInfo manager, FMLProxyPacket packet, EntityPlayer player );
-}
+	public PacketLogger( String side )
+	{
+		super( LogLevel.INFO );
+
+		this.side = side;
+	}
+
+	@Override
+	protected String formatMessage( String eventName, Object msg )
+	{
+		if( msg instanceof AppEngPacket )
+		{
+			AppEngPacket packet = (AppEngPacket) msg;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'packet' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)